### ISSUE TITLE
[Bug] Unexpected datetime.timedelta object being put through from_js_time() function

### ISSUE THREAD

**Issue by caraar12345** [AngellusMortis/pyunifiprotect#252]
**I know I'm using an Early Access build of UniFi Protect, but it's worked fine up until I upgraded Home Assistant to 2022.12.0b1 this morning**.

Somehow, a `datetime.timedelta` object is making its way through to the `from_js_time()` function in `/usr/local/lib/python3.10/site-packages/pyunifiprotect/utils.py`.

- UniFi console: **UniFi Dream Router**
- UniFi OS: **UDR v3.0.13**
- UniFi Protect: **2.7.5**
- Home Assistant: **2022.12.0b1** 

<details>
<summary>Traceback:</summary>

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/unifiprotect/__init__.py", line 93, in async_setup_entry
    await _async_setup_entry(hass, entry, data_service)
  File "/usr/src/homeassistant/homeassistant/components/unifiprotect/__init__.py", line 125, in _async_setup_entry
    await data_service.async_setup()
  File "/usr/src/homeassistant/homeassistant/components/unifiprotect/data.py", line 109, in async_setup
    await self.async_refresh()
  File "/usr/src/homeassistant/homeassistant/components/unifiprotect/data.py", line 129, in async_refresh
    updates = await self.api.update(force=force)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/api.py", line 596, in update
    self._bootstrap = await self.get_bootstrap()
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/api.py", line 772, in get_bootstrap
    return Bootstrap.from_unifi_dict(**data, api=self)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 115, in from_unifi_dict
    data = cls.unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/bootstrap.py", line 190, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 346, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj_dict(data[key], unifi_dicts[key], api)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 292, in _clean_protect_obj_dict
    items[key] = cls._clean_protect_obj(value, klass, api)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 276, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/devices.py", line 835, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 752, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/data/base.py", line 329, in unifi_dict_to_dict
    data[key] = convert_unifi_data(data[key], cls.__fields__[key])
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/utils.py", line 199, in convert_unifi_data
    value = from_js_time(value)
  File "/usr/local/lib/python3.10/site-packages/pyunifiprotect/utils.py", line 131, in from_js_time
    return datetime.fromtimestamp(int(num) / 1000, tz=timezone.utc)
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'datetime.timedelta'
```

</details>

**Comment by AngellusMortis** [comment]
Already fixed in 4.5.2

**Comment by caraar12345** [comment]
Oh fantastic! Thanks 😄
